index.html

```diff
@@ -31,26 +26,25 @@
 					</li>
 				</ul>
 			</div>
+			<div class="search-bar input">
+				<input type="text" />
+				<button>
```

Comment:
try input type : "search".

Reply:
I didn't know that such a type is there thank u!